### ISSUE TITLE
Request to add Spyder-ide 

### ISSUE THREAD

**Issue by SoleymanOfficial** [sonph/onehalf#37]
I love onehalf color schemes but as I switched over spyder from sublime, how could I use onehalf (lighter version) on spyder ?

**Comment by sonph** [comment]
Glad to hear you like it @Sol3y! I've never used Spyder myself, but if you can find a color scheme file, I can help modify it to use the light colors.

**Comment by Sol3y** [comment]
@sonph here's few Spyder color schemes https://github.com/joonro/Spyder-Color-Themes as far I guess 16 elements .

Below is the screenshot of Python IDLE's default color scheme currently I'm using in Spyder. 

![idle](https://user-images.githubusercontent.com/13655344/42985237-110eb4a8-8be0-11e8-9e7d-669c1a7246f2.PNG)

**Comment by sonph** [comment]
I took a quick look at one of the example color scheme and put in the colors: [onehalf-light.ini](https://gist.github.com/sonph/17695389fb33d5589ef9f79d1d9072d0).

Can you follow the instructions on https://github.com/joonro/Spyder-Color-Themes to test this out and perhaps take a screenshot? I'm sure there are places we'll need to change stuffs

**Comment by Sol3y** [comment]
Looks cool, thanks for the effort @sonph I really appreciate your time :)

**Left**: Sublime Text 3 with Onehalf light color scheme
**Right**: Spyder 4 with Onehalf light color scheme

![onehalf](https://user-images.githubusercontent.com/13655344/43046074-630746a6-8db3-11e8-8143-a0592bed8acc.PNG)

**Comment by sonph** [comment]
No problem :) but a couple things:

1. The colors in your sublime text screenshot look off. I mean the strings `'Driving'` and `'Updating software'` have different colors. What Sublime Text version are you using? On my machine it looks like this (ST 3.1.1)
<img width="590" alt="screen shot 2018-07-22 at 11 24 15 am" src="https://user-images.githubusercontent.com/2466660/43048786-1b8475a8-8da2-11e8-936d-a98b18e11e83.png"> which looks a bit more consistent.

2. If you're wondering about the differences in colors (e.g. `def drive` has a different color in each one), that's because each IDE uses its own syntax parsing engine.

3. I notice the italic `self` is clipping into `)` (line 6, 7 and 12 in spyder), so I reverted that. See latest [onehalf-light.ini](https://gist.github.com/sonph/17695389fb33d5589ef9f79d1d9072d0)

**Comment by Sol3y** [comment]
You're right @sonph my color scheme was broken -_- by a 3rd party package (Colorcoder), I've fixed it tho.

![onehalf](https://user-images.githubusercontent.com/13655344/43049239-1ee3df44-8e16-11e8-8cdc-f9a66cc95d2d.PNG)

Also the latest one looks nicer in spyder ;).

Thanks a lot for your time @sonph :)
